Code/Tools/FBuild/FBuildCore/Graph/LinkerNode.cpp

```diff
@@ -612,7 +612,7 @@ void LinkerNode::GetAssemblyResourceFiles( Args & fullArgs, const AString & pre,
         for ( const AString * it=tokens.Begin(); it!=end; ++it )
         {
             const AString & token = *it;
-            if ( ( token == "-shared" ) || ( token == "-dynamiclib" ) || ( token == "--oformat=prx" ) )
+            if ( ( token == "-shared" ) || ( token == "-dynamiclib" ) || ( token == "--oformat=prx" ) || ( token == "-Wl,--oformat=prx" ) )
             {
```

Comment:
Seems like this would miss cases like:
```C++
-Wl,--someotherthing,--oformat=prx
```
How about:
```C++
|| ( token.BeginsWith( "-Wl" ) && token.Find( "--oformat=prx" ) )
```